### ISSUE TITLE
Meet this weekend

### ISSUE THREAD

**Issue by NobleKennamer** [NobleKennamer/astro_porject#14]
@arielkruger1989 @abhisaarsharma @dcockerham 

When can people meet this weekend. preferably Saturday.

Just post anytime on here.

I'm free all day

**Comment by arielkruger1989** [comment]
I can meet anytime after 1pm.

**Comment by abhisaarsharma** [comment]
I might have some work out of town this weekend..

**Comment by dcockerham** [comment]
I can meet any time this weekend.

**Comment by NobleKennamer** [comment]
Alright let's meet at 2:00pm on Saturday.

I suggest meeting in front of the building that my office is in.

Does this work for everyone?

**Comment by arielkruger1989** [comment]
Yeah. Let's do this. Just let us know your phone number just in case. My phone number is (949)501-2507

**Comment by dcockerham** [comment]
2pm Saturday works for me.  My phone is (808) 652-5004.
